### DISCUSSION TITLE
Bug: script result is not displayed

### DISCUSSION THREAD

**Discussion by shoksdev** [netbox-community/netbox#17285] [Help Wanted!] [answered]

**Comment by candlerb** [comment]
Not enough information there to reproduce. At a minimum you would need to provide: Netbox version, OS version, and browser version; exactly what URL you are accessing; exactly what script you are running; the exact sequence of steps to reproduce the problem.

**Reply by candlerb** [reply, marked as answer]
I don't know what v0.21.0 refers to.

Netbox v3.1.11 is more than two years old, so your problem is unlikely to be reproduced - and no fixes are backported to older Netbox versions anyway.

There have certainly been changes to how script results are displayed since then, for example #13097.

I suggest you set up a test system with v4.0.10 and see if you can reproduce it there. If you can, then raise a bug report with full reproduction details (including a script which demonstrates the problem).

**Reply by shoksdev** [reply]
Thank you, have a good day!

**Reply by candlerb** [reply]
For future reference: the poster has deleted both their original messages, so the replies I made make no sense any more. This is annoying and disrespectful.

**Reply by shoksdev** [reply]
I'm so sorry, All the best to you!
